css/main.css

```diff
@@ -6,3 +6,147 @@ Green: #00B172
 White: #FFF
 
 */
+h1, body, html, main {
+    margin: 0;
+  }
+  /*Import the font and set the default size*/
+  html {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px
+  }
+  
+  body {
+    background-color: white;
+    font-family: "Time new Roman", sans-serif, serif;
+  }
+
+
+  header > nav{
```

Comment:
No need to use the tag '>', since the nav is a child of the header, it will apply it with a simple space between 'header' and 'nav'.
The same for all the code below